### DISCUSSION TITLE
Which voice is the best for natural flow?

### DISCUSSION THREAD

**Discussion by Connor56** [rany2/edge-tts#340] [General]
There are so many options in Edge TTS that I'm suffering from its success! Thoroughly trying all the voices is a huge time sink, so I'd rather put it to the community. Do you have a favourite voice? Why?

I'll start, my favourite is: GB-RyanNeural.

I like the voice because it sounds the least robotic in the sounds it produces. However, it does suffer from unnatural-sounding emphasis on words.

**Comment by rany2** [comment]
I personally like `en-US-EmmaMultilingualNeural`.

**Reply by Connor56** [reply]
Awesome, thank you!

**Comment by mrfragger** [comment]
en-US-AndrewMultilingualNeural en-US-AriaNeural en-US-BrianMultilingualNeural en-US-AvaMultilingualNeural en-US-ChristopherNeural en-CA-ClaraNeural en-US-EricNeural en-US-EmmaMultilingualNeural en-US-GuyNeural en-US-JennyNeural en-US-RogerNeural en-US-MichelleNeural en-GB-RyanNeural en-GB-LibbyNeural en-US-SteffanNeural en-GB-SoniaNeural

those are my 16 favorites

**Reply by Beklem** [reply]
I think en-US-RogerNeural is the best here!

**Comment by josediaznunez** [comment]
my two favorites are
en-US-BrianMultilingualNeural   
en-US-AndrewMultilingualNeural

**Comment by yeartwothousandfive** [comment]
en-PH-RosaMultilingualNeural

**Reply by Beklem** [reply]
btw this one doesn't work!

**Reply by Mehedi7978** [reply]
> btw this one doesn't work!

you are right !

**Reply by bigtajine** [reply]
Just to note it's en-PH-RosaNeural

**Comment by EdAn-EdAn** [comment]
en-IE-EmilyNeural
